lib/transformers/default.js

```diff
@@ -63,7 +63,10 @@ module.exports = ({
         return govukCheckboxes(args);
     }
 
-    if (schema.type === 'object') {
+    if (schema.type === 'object' && 'properties' in schema) {
```

Comment:
move `'properties' in schema` to the next condition so it is only checked for the summary schema

Reply:
@armoj Done!